src/components/films/components/filmsHeader/FilmsNav.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { useState } from "react";
+import { FilmsItem, FilmsNav } from "./filmsHeader.styled";
+
+const FilmsNavC = () => {
+  const [ categories ] = useState(['all',  'documentary', 'comedy', 'horror', 'crime']);
```

Comment:
You're not supposed to hardcode these within your component, but rather receive as props for the sake of component re-usability.